### PR TITLE
Prints fixes for rc/2022-july

### PR DESCRIPTION
**Pull request by iulianpascalau**
## Description of the reasoning behind the pull request (what feature was missing / how the problem was manifesting itself / what was the motive behind the refactoring)
- added more prints in the level DB constructors
- handled some log.Warn and log.Error prints when encountering the `DB is closed` error
- health records parameters adjustments
  
## Proposed Changes
- added prints to determine how much time it takes for the constructor to finish
- added better handling

## Testing procedure
- standard internal testnets procedure

### CODE REVIEW

File: storage/leveldb/leveldb.go [hunk 1]
```diff
@@ -80,7 +88,11 @@ func NewDB(path string, batchDelaySeconds int, maxBatchSize int, maxOpenFiles in
 	})
 
 	crtCounter := atomic.AddUint32(&loggingDBCounter, 1)
-	log.Debug("opened level db persister", "path", path, "created pointer", fmt.Sprintf("%p", bldb.db), "global db counter", crtCounter)
+	sw.Stop("NewSerialDB")
```
Comment (BeniaminDrasovean) [inline storage/leveldb/leveldb.go:91]: "NewDB" is started, but "NewSerialDB" is stopped. Maybe use a variable to be sure the same sw is started and stopped

**Review by BeniaminDrasovean — CHANGES_REQUESTED** [review]

File: sharding/nodesCoordinator/indexHashedNodesCoordinator.go [hunk 2]
```diff
@@ -634,7 +635,11 @@ func (ihnc *indexHashedNodesCoordinator) EpochStartPrepare(metaHdr data.HeaderHa
 	ihnc.fillPublicKeyToValidatorMap()
 	err = ihnc.saveState(randomness)
 	if err != nil {
-		log.Error("saving nodes coordinator config failed", "error", err.Error())
+		logLevel := logger.LogError
```
[the remark this answers is not in the document]
Comment (iulianpascalau) [reply sharding/nodesCoordinator/indexHashedNodesCoordinator.go:638]: done

File: storage/leveldb/leveldb.go (hunk 1 above)
Comment (iulianpascalau) [reply storage/leveldb/leveldb.go:91]: good catch, refactored.

**Review by BeniaminDrasovean — APPROVED** [review]

File: sharding/nodesCoordinator/indexHashedNodesCoordinator.go [hunk 3]
```diff
@@ -256,23 +257,23 @@ func (ihnc *indexHashedNodesCoordinator) setNodesPerShards(
 	}
 
 	var err error
-	var isValidator bool
+	var currentNodeIsValidator bool
```
Comment (dragos-rebegea) [inline sharding/nodesCoordinator/indexHashedNodesCoordinator.go:260]:
```suggestion
	var isCurrentNodeValidator bool
```

File: storage/leveldb/leveldb.go [hunk 4]
```diff
@@ -20,6 +21,8 @@ var _ storage.Persister = (*DB)(nil)
 
 // read + write + execute for owner only
 const rwxOwner = 0700
+const mkdirAllFunction = "MkdirAll"
```
Comment (dragos-rebegea) [inline storage/leveldb/leveldb.go:24]:
```suggestion
const mkdirAllFunction = "mkdirAll"
```
to start with lowercase, the same as openLevelDBFunction

**Review by dragos-rebegea — DISMISSED** [review]

File: storage/leveldb/leveldb.go (hunk 4 above)
Comment (iulianpascalau) [reply storage/leveldb/leveldb.go:24]: done

File: sharding/nodesCoordinator/indexHashedNodesCoordinator.go (hunk 3 above)
Comment (iulianpascalau) [reply sharding/nodesCoordinator/indexHashedNodesCoordinator.go:260]: done

**Review by dragos-rebegea — APPROVED** [review]

**Review by BeniaminDrasovean — APPROVED** [review]

**Review by gabi-vuls — APPROVED** [review]
System test passed.

**Pull request merged**